src/app/views/home.js

```diff
@@ -25,6 +25,8 @@ var HomeView = Backbone.View.extend({
     this.model.fetch();
     this.model.on('change', this.render);
 
+    this.model.set({'title': 'The Detroit Ledger'});
```

Comment:
For the homepage, it might be good to load some keywords. like "The Detroit Ledger - information about grants and nonprofits in Detroit" or "The Detroit Ledger - Detroit grant and nonprofit data"